fluent-langneg/src/negotiate_languages.ts

```diff
@@ -1,23 +1,8 @@
-import filterMatches from "./matches";
+import {filterMatches} from "./matches";
 
-function GetOption(options, property, type, values, fallback) {
```

Comment:
I removed this function because I don't think the implementation of `@fluent/langneg` needs to be 100% defensive. For TypeScript consumers, TS already gives us some of those guaranties. For non-TS consumers, we don't usually verify all inputs in other `@fluent` packages as thoroughly as this function did it.